src/scenarios/shared/runner.py

```diff
@@ -957,3 +966,10 @@ def run(self):
             if not (self.dirs or builtdir):
                 raise Exception("Dirs was not passed in and neither %s nor %s exist" % (const.PUBDIR, const.BINDIR))
             sod.runtests(scenarioname=self.scenarioname, dirs=self.dirs or builtdir, upload_to_perflab_container=self.upload_to_perflab_container, artifact=self.traits.artifact)
+
+        elif self.testtype == const.BUILDTIME:
+            startup = StartupWrapper()
+            if not (self.binlogpath):
+                raise Exception("Binlog path was not passed provided.")
```

Comment:
Should we check if the path exists? 
Additionally, the exception message sounds wrong, maybe something like:
```py
raise Exception("For build time measurements a valid binlog path must be provided")
```